tests/spread/integration/util-linux/task.yaml

```
@@ -0,0 +1,148 @@
+summary: Integration tests for util-linux
+
+systems:
+  - -ubuntu-24.04-ppc64le
+  - -ubuntu-24.04-s390x
+
+environment:
+  SLICE/bins: "bins"
+  SLICE/su: "su-support"
+
+execute: |
+  rootfs="$(install-slices util-linux_${SLICE} base-files_base bash_bins)"
+
+  # we need dev/sys mounted for some of them
+  mkdir "${rootfs}"/dev
+  mkdir "${rootfs}"/proc
+  mkdir "${rootfs}"/sys
+
+  mount --rbind /dev "${rootfs}"/dev
+  mount --rbind /proc "${rootfs}"/proc
+  mount --rbind /sys "${rootfs}"/sys
+
+  case ${SLICE} in
+    bins)
+      # Binaries involving disk, partition and device operations are only tested with
+      # the `--help` command as the smoke test.
+      # Some of the binaries are not possible to be tested with their functionalities
+      # due to the limitation of the spread test environment.
+      # Some of the binaries neither changes the status of the system, nor have
+      # deterministic outputs across different systems. They are tested without
+      # assertions.
+      chroot "${rootfs}" addpart --help
+      chroot "${rootfs}" choom -p 1
+      chroot "${rootfs}" chrt -p 1
+      chroot "${rootfs}" delpart --help
+      # Accessing kernel dmesg is not permitted in LXD.
+      chroot "${rootfs}" dmesg --help
+      chroot "${rootfs}" fallocate -l 1M /tmp/fallocate-test
+      test -f "${rootfs}"/tmp/fallocate-test
+      chroot "${rootfs}" findmnt | grep "/dev"
+      chroot "${rootfs}" flock /tmp/bash.lock -c "exit 0"
+      chroot "${rootfs}" getopt Hello world | grep " -- world"
+      mkdir -p "${rootfs}"/tmp/hardlink-test
+      echo "Hello world" > "${rootfs}"/tmp/hardlink-test/file1
+      cp "${rootfs}"/tmp/hardlink-test/file1 "${rootfs}"/tmp/hardlink-test/file2
+      chroot "${rootfs}" hardlink /tmp/hardlink-test/ | grep 'Saved:[[:blank:]]*12 B'
+      chroot "${rootfs}" ionice -P 1 | grep "best-effort: prio"
```

```suggestion
      chroot "${rootfs}" ionice -P 1 | grep "prio"
```

I believe this is the last chip of the puzzle...